script/Ciel/Day 15/15_01A_CIEL15A_1.txt

```diff
@@ -978,7 +1007,7 @@ What do you know...
 [sha:307def57147fbde830297b3c5d70a09d93abe2f9]{
 -- Page 58, Offset 41561.
 -- 森の中に、こんな和風の屋敷があったなんて。
-Who'd have thought there'd be a Japanese-style building like this in the middle of the forest.
+Who'd have thought there'd be a traditional mansion like this in the middle of the forest.
```

Comment:
I feel like mansion is pretty strongly western-coded to me, I probably wouldn't refer to a minka that way.